app/Http/Controllers/LockerController.php

```diff
@@ -27,7 +28,7 @@ public function index()
      */
     public function store()
     {
-        $locker = Locker::create(['guid' => substr(md5(rand()), 0, 8)]);
+        $locker = Locker::create(['guid' => Str::random(8)]);
```

Comment:
I think it's best practice to encapsulate this GUID generation inside the model. It can be done using a [model observer](https://laravel.com/docs/6.x/eloquent#observers); acting based on a `creating` event that automatically gets triggered by Laravel.